src/database/migrations/1676872272236-email.ts

```diff
@@ -0,0 +1,11 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
```

Comment:
Se não for usar podemos deletar

Reply:
concordo, irei passar para o Amaro.
Essas alterações entraram na minha PR pois, mergeamos a minha branch na dele, e como havia conflitos acabou que tudo o que ele estava fazendo veio para a minha.

POrem esses pontos serão corrigidos na proximo PR que ele irá enviar